src/python_framework_v02/troute/nhd_io.py

```diff
@@ -53,6 +53,7 @@ def read_custom_input(custom_input_file):
     restart_parameters = data.get("restart_parameters", {})
     output_parameters = data.get("output_parameters", {})
     run_parameters = data.get("run_parameters", {})
+    parity_parameters = data.get("parity_parameters", {})
     # TODO: add error trapping for potentially missing files
     return (
```

Comment:
You might think about wrapping these return parameters in a dataclass or namedtuple. It will help you keep things organized.

Reply:
will address in #189